lib/baseClasses/HttpError.js

```diff
@@ -22,21 +22,26 @@ function HttpError() {
 
     // normalize the signatures
     var parsed = helpers.parseVariadicArgs(arguments, true);
-    var options = parsed.options;
     var args = parsed.args;
+    var options = parsed.options || {};
 
-    // inherit from WError, call super first before doing anything else!
+    // inherit from WError, call super first before doing anything else!  WError
+    // will handle calling captureStackTrace, using arguments.callee to
+    // eliminate unnecessary stack frames.
     WError.apply(self, args);
 
     /**
      * the error message.
      * when message isn't passed in with options, expect WError constructor
      * to fill in the message for us with the passed through args,
-     * usually via sprintf or just straight strings.
+     * usually via sprintf or just straight strings. thus, only set it
```

Comment:
This seems like a bug with verror right? where if we set a message that's already been set, verror blows up? seems like a bug we should file with verror.